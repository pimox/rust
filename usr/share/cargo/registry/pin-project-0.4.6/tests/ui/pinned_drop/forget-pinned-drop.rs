use pin_project::pin_project;

#[pin_project(PinnedDrop)] //~ ERROR E0277
pub struct Foo {
    #[pin]
    field: u8,
}

fn main() {}
