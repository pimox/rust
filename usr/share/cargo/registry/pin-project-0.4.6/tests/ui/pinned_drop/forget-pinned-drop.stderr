error[E0277]: the trait bound `Foo: pin_project::__private::PinnedDrop` is not satisfied
 --> $DIR/forget-pinned-drop.rs:3:15
  |
3 | #[pin_project(PinnedDrop)] //~ ERROR E0277
  |               ^^^^^^^^^^ the trait `pin_project::__private::PinnedDrop` is not implemented for `Foo`
  |
  = note: required by `pin_project::__private::PinnedDrop::drop`
